models/intel/device_support.md

```diff
@@ -72,6 +72,9 @@
 | person-detection-0201 | YES | YES | YES |
 | person-detection-0202 | YES | YES | YES |
 | person-detection-0203 | YES | YES | YES |
+| person-detection-0301 | YES | YES | YES |
```

Comment:
have you really tested on GPU and MyriadX ? If not please not put `YES`, we will update based on accuracy validation later. Notice, working on CPU is a must (one of the release criteria)

Reply:
No, did not test. Ok, I will put `NO`.

Reply:
leave just empty cell in the table. We do not put NO explicitly

Reply:
Removed untested devices.